### DISCUSSION TITLE
Welcome to flutter_staggered_grid_view Discussions!

### DISCUSSION THREAD

**Discussion by letsar** [letsar/flutter_staggered_grid_view#209] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by subrotoxing** [comment]
Hi.... I've been trying to look up for the documentation but I can't find how to work with count builder ? I have this code written with 0.4.1 .. how can I work this for the latest version ? Thanks before.

```
SliverStaggeredGrid.countBuilder(
                crossAxisCount: 1,
                staggeredTileBuilder: (c)=> const StaggeredTile.fit(1),
                itemBuilder: (context,index)
                {
                 
                  return SomeWidget(
                    index: index,
                    context: context,
                  );
                },
                itemCount: snapshot.data!.docs.length,
              );
```

**Comment by Pierre-Monier** [comment]
Hi, I'm a developer from France, focusing on developing Flutter application. I first use this library in one of my company project, and it works just as expected. Really like this library. One suggestion I made in [this issue ](https://github.com/letsar/flutter_staggered_grid_view/issues/225)  and I wanted to know what do you think about it ?
